pkg/controller/che/che_controller.go

```diff
@@ -580,6 +581,24 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 		}
 	}
 
+	if len(instance.Spec.Server.CheClusterRoles) > 0 {
+		cheClusterRoles := strings.Split(instance.Spec.Server.CheClusterRoles, ",")
+		for _, cheClusterRole := range cheClusterRoles {
+			cheClusterRole := strings.TrimSpace(cheClusterRole)
+			cheClusterRoleBindingName := "che-" + cheClusterRole
```

Comment:
Pls use CRB name like this `che-<clusterrole>-<namespace>` then we can distinguish CRB created for different namesaces and would be able to remove on chectl with `server:delete` command.

Reply:
done